ting.admin.inc

```diff
@@ -162,6 +162,22 @@ function ting_admin_ranking_settings($form, &$form_state) {
     ),
   );
 
+
+  $ting_sort_value = variable_get('ting_sort_default', 'rank_frequency');
+  $form['ting_sort_default'] = array(
+    '#title' => t('Default sort method (best match)'),
+    '#type' => 'fieldset',
+    '#prefix' => '<div id="sort-default-fields-wrapper">',
+    '#suffix' => '</div>',
+    '#tree' => TRUE,
+  );
+
+  $form['ting_sort_default']['fields'] = array(
+    '#type' => 'ting_sort_default',
```

Comment:
That pattern is used for `ting_ranking_field` which requires [significant processing](https://github.com/pjohans/ting/blob/Issue1113/ting.module#L843).

It is not done that way for `ting_ranking_custom` which is simply a `checkbox`. This is a trivial field and thus it is unneeded as well.